src/controllers/csi/provisioner/controller.go

```diff
@@ -94,6 +95,11 @@ func (provisioner *OneAgentProvisioner) Reconcile(ctx context.Context, request r
 		return reconcile.Result{RequeueAfter: longRequeueDuration}, provisioner.db.DeleteDynakube(ctx, request.Name)
 	}
 
+	if dk.CodeModulesImage() != "" && dk.CodeModulesVersion() != "" {
+		log.Info("dynakube status is not yet ready, requeuing", "dynakube", dk.Name)
```

Comment:
wouldn't either the hash (for code modules image) or the version be an indication that the status is ready? i.e. depending on the "source"